tiers/checks.go

```diff
@@ -308,6 +311,17 @@ func networkPolicyIngress(name, group string) namespaceObjectsCheck {
 	}
 }
 
+func idlers(namespaceTypes ...string) clusterObjectsCheck {
+	return func(t *testing.T, memberAwait *wait.MemberAwaitility, userName string) {
+		for _, nt := range namespaceTypes {
+			idler, err := memberAwait.WaitForIdler(fmt.Sprintf("%s-%s", userName, nt))
+			require.NoError(t, err)
+			assert.Equal(t, userName, idler.ObjectMeta.Labels["toolchain.dev.openshift.com/owner"])
+			assert.Equal(t, int32(28800), idler.Spec.TimeoutSeconds)
+		}
```

Comment:
Correct me if I'm wrong, but you don't verify the number of the `Idler` objects anywhere. 
We should check that to verify the case when a user is promoted to `team` tier, then the controller should remove the redundant `*-code` Idler object

Reply:
Good point. Added it here: https://github.com/codeready-toolchain/toolchain-e2e/pull/135/commits/833004e32de39c1f40995cde2000f12f77094283